### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by robinkrahl** [d-e-s-o/nitrocli#33]
Update the documentation (README + man page) for the 0.2.0 release:
- [x] `config`
- [x] `otp`
- [x] `pws`
- [x] `storage`
- [x] general

**Comment by d-e-s-o** [comment]
Thanks for raising this issue for tracking documentation updates and for stepping up and fixing the documentation @robinkrahl :)
I double checked the individual items and could not find anything lacking.

**Comment by d-e-s-o** [comment]
Yeah, I wanted to make a couple more improvements, but you raise valid points. Let's reopen since a new issue would just end up having pretty much the same title anyway.

I agree that having an HTML man page would be nice. The source of truth should be in this repository, where it is mirrored I don't care so much about. But yes, Github pages needs an `index.html` in the root from what I understand, which is not going to fly. We'll find a solution.

**Comment by d-e-s-o** [comment]
@robinkrahl I experimented a bit with proper rendering of the `man` page. Both PDF and HTML are easily possible. HTML as created by groff doesn't look nice I'd say and we can potentially find a better solution, but the PDF file is not the worst.

Funny thing is that while I can't directly link the HTML file (without Github pages or some third party service; at least not that I am aware of), Github actually seems to display PDFs inline just fine.
I added a link [here](https://github.com/d-e-s-o/nitrocli/tree/topic/documentation/nitrocli#usage) to the README (bottom of the `Usage` section). It's not the greatest, but it's nice that:
- it's a relative link (i.e., no hard coding of github.com)
- it makes it clear that the PDF is part of the repository (by having the usual UI stuff surround it)

(now this is unlikely to work on, say, `crates.io` it just messed up all relative links, but I can live with that)

**Comment by robinkrahl** [comment]
@d-e-s-o We forgot to fix the most obvious – the first line in the README:
> nitrocli is a program that provides a command line interface for certain commands on the Nitrokey Storage device.

Could you please fix that with the next release?

**Comment by d-e-s-o** [comment]
Will do. Thanks for pointing that out.
